graphene_linked_events/tests/test_api.py

```diff
@@ -1118,6 +1118,87 @@ def test_publish_event(
     snapshot.assert_match(executed)
 
 
+UNPUBLISH_EVENT_MUTATION = """
+mutation unpublishEvent($input: PublishEventMutationInput!){
+  unpublishEventMutation(event: $input){
+    response{
+      statusCode
+      body {
+        id
+        startTime
+        endTime
+        publicationStatus
+      }
+    }
+  }
+}
+"""
+
+
+def test_unpublish_event_unauthorized(
+    snapshot,
+    api_client,
+    user_api_client,
+    staff_api_client,
+    mock_update_event_data,
+    organisation,
+    person,
+):
+    # Reuse update event variables
+    variables = deepcopy(UPDATE_EVENT_VARIABLES)
+    del variables["input"]["draft"]
+    executed = api_client.execute(UNPUBLISH_EVENT_MUTATION, variables=variables)
+    assert_permission_denied(executed)
+    executed = user_api_client.execute(UNPUBLISH_EVENT_MUTATION, variables=variables)
+    assert_permission_denied(executed)
+
+    variables = deepcopy(UPDATE_EVENT_VARIABLES)
+    del variables["input"]["draft"]
```

Comment:
Isn't the same have been already done on lines 1148 and 1149? 🙂

Reply:
Must be leftover from the old `test_publish_event`, then copied here :D fixed